include/oxen/quic/address.hpp

```diff
@@ -340,6 +340,10 @@ namespace oxen::quic
             return *this;
         }
 
+        bool operator==(const Path& other) const { return std::tie(local, remote) == std::tie(other.local, other.remote); }
```

Comment:
Might be worth it/better to do `operator<=>` instead?

Reply:
great point!

Reply:
but will define spaceship separately and invoke it within `operator==`, just to follow the Address  pattern